grapple/types/streamfield.py

```diff
@@ -304,12 +303,7 @@ class Meta:
         interfaces = (StreamFieldInterface,)
 
     def resolve_value(self, info, **kwargs):
-        # Allow custom markup for RichText
-        if grapple_settings.RICHTEXT_FORMAT == "html":
-            return render_to_string(
-                "wagtailcore/richtext.html", {"html": expand_db_html(self.value.source)}
-            )
-        return self.value.source
+        return RichTextType.serialize(self.value.source)
```

Comment:
was going to suggest that we change the definition of the `RichTextBlock` object type
to have `value = RichTextType(required=True)`, but value is a dict in the form of `{"source": "raw value"}`, so that would've needed handling ing the `RichText` scalar definition which is not as neat. so 👍